### ISSUE TITLE
telewizjaonline.pl

### ISSUE THREAD

**Issue by UncleHoSzczecin** [mbebe/blomqvist#815]
Czy możesz rzucić łaskawym okiem na tego usługodawcę?
Pół roku za 20 zł to ciekawa oferta :)

**Comment by mtr81** [comment]
A masz jakiś płatny pakiet, żebyś mógł sprawdzić czy wtyczka działa ze wszystkimi kanałami? Napisałem ją 4 miesiące temu, kiedy były dostępne tylko bezpłatne...

**Comment by UncleHoSzczecin** [comment]
Mam. Możesz podrzucić wtyczkę i przetestuję.

**Comment by mtr81** [comment]
Udostępniłem ci na GH. Jak będzie OK - upubliczniam.

**Comment by UncleHoSzczecin** [comment]
Działa perfekcyjnie po zalogowaniu. Dzięki wielkie!
Świetnie że dobiera automatycznie najlepszą jakość.
Jesteś Wielki!
P.s. Gdyby udało się jeszcze dorzucić generator m3u to będzie już bajecznie :)

**Comment by UncleHoSzczecin** [comment]
Dokupiłem celowo dodatkową usługę żeby sprawdzić czy aktualizuje się i jest ok ale...
Brakuje tylko automatycznego logowania wraz ze startem KODI. Skróty w ulubionych przez to odpadają.
Taki drobiazg a by ucieszył :)

**Comment by mtr81** [comment]
Ogarnę temat.

**Comment by mtr81** [comment]
Poszły poprawki na GH. Wprowadziłem automatyczne przywrócenie logowania, gdy podczas ostatniego użytkowania wtyczki użytkownik posiadał taki status. Z kolei lista M3U jest możliwa do wygenerowania tylko w sytuacji, gdy użytkownik jest zalogowany lub posiada status gościa. Prośba o przetestowanie i feedback.

**Comment by UncleHoSzczecin** [comment]
Niestety zarówno poprzednia jak i obecna wersja wtyczki nie działają.
Wywala jedynie komunikat o błędnych danych logowania TVO.
Przez web i na telefonie w appce działają dane logowania poprawnie.
Komunikat zaczął wyskakiwać zanim zauważyłem wiadomość od Ciebie więc zaszło coś niezwiązanego z nową wersją.

**Comment by UncleHoSzczecin** [comment]
A. I nie odtwarza również z konta gościa.

**Comment by UncleHoSzczecin** [comment]
Cholera. Zaczekajmy do jutra. Może coś im się wysypało albo... Nie wiem...
A z web nie da rady wyciągnąć streamów?
Zupełnie się nie znam na tworzeniu wtyczek.

**Comment by mtr81** [comment]
Teraz to już nie wiem o co Ci chodzi. Chcesz, żebym podał Ci linki do manifestów M3U?

**Comment by UncleHoSzczecin** [comment]
Obawiam się że nic mi one nie powiedzą. Niestety nie znam się na tym.

**Comment by mtr81** [comment]
Zerknę do wtyczki i ich API i zobaczę co się podziało.

**Comment by UncleHoSzczecin** [comment]
Trzymam kciuki!

**Comment by mtr81** [comment]
Już widzę co się stało -> 'g-recaptcha-response' w POST data przy logowaniu.

**Comment by UncleHoSzczecin** [comment]
Pacjent będzie żył? Da się coś z tym zrobić?

**Comment by mtr81** [comment]
Muszę zerknąć jak to wygląda w apce na Androida, bo obecna wtyczka powstała w oparciu o API na www.

**Comment by mtr81** [comment]
@UncleHoSzczecin
Spróbuj: GH -> telewizjaonline2. Instalacja nie przykryje poprzedniej wersji opartej na www.

**Comment by UncleHoSzczecin** [comment]
Działa a sprawdziłem na KODI 20 Alpha nawet.
M3U wygenerowane. Logowanie...
No właśnie...
Nigdzie nie podawałem loginu i hasła ani nie widzę opcji w ustawieniach.
Pobrało ze starszej wersji?

**Comment by mtr81** [comment]
Cuda na kiju panie ;-)

**Comment by UncleHoSzczecin** [comment]
No to pięknie :D Gratulacje, podziękowania i prośba o umieszczenie w repo :)

**Comment by mtr81** [comment]
A pakiet stacji ci się zgadza?

**Comment by UncleHoSzczecin** [comment]
Tak. Są nawet TVN24 za które siłą rzeczy dopłaciłem 7 zł za miesiąc :D
Zdeinstalowałem obie wtyczki, zrestartowałem KODI i ponownie zainstalowałem jedynie wersję 2.
Działa.

**Comment by UncleHoSzczecin** [comment]
Tak na marginesie:
PolstreamLive to Twoje dzieło?
Bo jeśli tak to też za nią wielkie dzięki ale czy jest możliwe i tam dorzucenie generatora m3u? :P

**Comment by mtr81** [comment]
W tego typu wtyczkach, ze względu na ich tymczasowy charakter, nie implementuję generatorów list.

**Comment by UncleHoSzczecin** [comment]
Ok. To logiczne. Dzięki za odpowiedź.

**Comment by Axis963** [comment]
@mtr81 jest szansa na opublikowanie wtyczki?

**Comment by CrowVip** [comment]
Może mi ktoś podesłać ta wtyczkę? Dziękuję.
